packages/ethereum-contracts/contracts/mocks/CustomSuperTokenMock.sol

```diff
@@ -63,4 +76,23 @@ contract CustomSuperTokenProxyMock is CustomSuperTokenBaseMock {
         // this makes msg.sender to self
         ISuperToken(address(this)).selfBurn(to, amount, userData);
     }
+    
+    // this function self calls transferFrom
```

Comment:
perhaps from grammatical reading perspective, holder should be the first parameter for "transfer from"